auth/server/src/main/java/com/seagates3/authorization/Authorizer.java

```diff
@@ -80,6 +81,45 @@ ServerResponse authorize(Requestor requestor,
     AuthorizationResponseGenerator responseGenerator =
         new AuthorizationResponseGenerator();
     String mainOperation = requestBody.get("S3Action");
```

Comment:
bool s3service = isS3Service(requestBody)  //returns true if operation requested is S3 Action

if(s3service) {
     // do bucket policy authorization
     // do anything else required for s3 service only like put policy opeartion
    // do ACL authorization
}
//common part
call IAMPolicy authorizer

Reply:
This is how it is rit now.

Reply:
Fixed the variable name from mainOperation -> s3Operation